src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java

```diff
@@ -81,7 +71,7 @@ public void isShootingToAlien(final Spaceship spaceship) {
      * from the Alien.
      *
      */
-    public void isShootingToSpaceship(final Alien alien) {
+    public void isShootingToSpaceship() {
```

Comment:
A better name would be isShootingToAlien

Reply:
The method is to say who "is shooting to Spaceship"? An Alien